### PR TITLE
Add flow with custom llm tool and flow CI

### PR DESCRIPTION
**Pull request by jiazengcindy**
# Description

Add flow with custom llm tool and flow CI

# All Promptflow Contribution checklist:
- [X] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by dorisjoy — DISMISSED** [review]

**Review by dorisjoy — DISMISSED** [review]

File: examples/tools/use-cases/custom_llm_tool_showcase/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Prerequisites
+
+Install promptflow sdk and other dependencies:
+```bash
+pip install -r requirements.txt
+```
+
+## Setup connection
+Prepare your Azure Open AI resource follow this [instruction](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal) and get your `api_key` if you don't have one.
```
Comment (16oeahr) [inline examples/tools/use-cases/custom_llm_tool_showcase/README.md:18]: why do we need to set up connection? Since line 7 says no

Comment (jiazengcindy) [reply examples/tools/use-cases/custom_llm_tool_showcase/README.md:18]: This is a fake connection, a similar flow is as following, thanks:
https://github.com/microsoft/promptflow/blob/404bce1467bc47f7e2732f9a020c43b3d48cd49c/examples/flows/standard/basic-with-connection/README.md?plain=1#L29
image

Comment (dorisjoy) [reply examples/tools/use-cases/custom_llm_tool_showcase/README.md:18]: Please refer the readme in this PR to refine the customconnection part https://github.com/microsoft/promptflow/pull/921

Comment (jiazengcindy) [reply examples/tools/use-cases/custom_llm_tool_showcase/README.md]: Changed, thanks.

**Review by 16oeahr — DISMISSED** [review]

**Review by dorisjoy — APPROVED** [review]

**Review by 16oeahr — APPROVED** [review]

**Pull request merged**